plugins/generic/externalFeed/ExternalFeedBlockPlugin.inc.php

```diff
@@ -36,62 +39,67 @@ function getHideManagement() {
 	 * its category.
 	 * @return String name of plugin
 	 */
-	function getName() {
+	public function getName() {
```

Comment:
I don't think the `getName` function is required anymore (the parent classes should provide a good enough implementation).